docs/unified-test-documentation/test-matrix.md

```diff
@@ -29,6 +31,12 @@ described in this documentation are dedicated to the given platform
 | 1.   | TPM Support                                       | TPM           | TPM001.001                           |
 | 2.   | Verified Boot support                             | VBO           | VBO001.001, VBO002.001, VBO003.001   |
 
+### Module: Dasharo performance
+
+| No.  | Supported test suite                              | Test suite ID | Supported test cases                 |
+|:-----|:--------------------------------------------------|:-------------:|:-------------------------------------|
+| 1.   | coreboot boot measure                             | CBMEM         | All                                  |
```

Comment:
This test case needs better name, in PCE we have coreboot booting time. Maybe `coreboot bring up time measurement` ?

Reply:
Okay, I will make the change.